server/api/getTwitchStreamers.post.ts

```diff
@@ -8,10 +8,9 @@ export default defineEventHandler(async (event) => {
     },
   };
   const streamers = body.streamersNames.join('&login=');
-  // console.log(streamers);
   const data = await fetch(
     `https://api.twitch.tv/helix/users?login=${streamers}`,
-    options,
+    options
```

Comment:
```suggestion
    options,
```
C'est normal qu'on ait les , en fin de tout, ça améliore la lecture, l'écriture et sur github aussi